vm/systemSmartContracts/esdt.go

```diff
@@ -407,10 +408,6 @@ func (e *esdt) toggleFreeze(args *vmcommon.ContractCallInput, builtInFunc string
 		e.eei.AddReturnMessage("cannot freeze")
 		return vmcommon.UserError
 	}
-	if len(args.Arguments[1]) != len(args.CallerAddr) {
```

Comment:
if len(args.Arguments) != 2 in line 398 and 426?

Reply:
done